backend/src/main/java/com/trackpack/app/TrackPackApplication.java

```diff
@@ -0,0 +1,13 @@
+package com.trackpack.app;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication(scanBasePackages = "com.trackpack.app")
```

Comment:
In future maybe you'll need to widen scope of scanning. As for now it's not a problem, so I'll merge your PR.